test/service.go

```diff
@@ -45,6 +45,15 @@ func CreateLatestService(logger *logging.BaseLogger, clients *Clients, names Res
 	return svc, err
 }
 
+// CreateLatestServiceWithResources creates a service in namespace with the name names.Service
+// that uses the image specified by imagePath
+func CreateLatestServiceWithResources(logger *logging.BaseLogger, clients *Clients, names ResourceNames, imagePath string) (*v1alpha1.Service, error) {
+	service := LatestServiceWithResources(ServingNamespace, names, imagePath)
+	LogResourceObject(logger, ResourceObjects{Service: service})
+	svc, err := clients.ServingClient.Services.Create(service)
+	return svc, err
```

Comment:
return clients.SC...

Reply:
Yeah, I noticed this in other ones as well, but opted to copy/paste.  Fixed.